package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "patios",
+  "version": "0.0.1",
+  "description": "Manage large Open API specifications and automatically generate TypeScript definitions from it.",
+  "main": "src/index.ts",
+  "type": "module",
+  "scripts": {
+    "test": "jest",
+    "dev": "ts-node-esm src/index.ts",
+    "format": "prettier -w .",
```

Comment:
If we're using both Prettier & eslint, should we consider being proactive in trying to avoid clashing rules?

This can be done using base configs like [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) and optional extra tools like [prettier-eslint](https://github.com/prettier/prettier-eslint).

https://prettier.io/docs/en/integrating-with-linters.html

Reply:
Added eslint-config-prettier and a script to package.json to run it, no conflicting rules reported yet